### PR TITLE
ARP: ExtremeTuxRacer.ExtremeTuxRacer 0.8.0.0

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23469)

### CODE REVIEW

File: manifests/e/ExtremeTuxRacer/ExtremeTuxRacer/0.8.0.0/ExtremeTuxRacer.ExtremeTuxRacer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ExtremeTuxRacer.ExtremeTuxRacer
+PackageVersion: 0.8.0.0
+PackageLocale: en-US
+Publisher: Extreme TuxRacer
+PublisherSupportUrl: https://sourceforge.net/projects/extremetuxracer/support
+PackageName: The ExtremeTuxRacer team
```
Comment (OfficialEsco) [inline manifests/e/ExtremeTuxRacer/ExtremeTuxRacer/0.8.0.0/ExtremeTuxRacer.ExtremeTuxRacer.locale.en-US.yaml:9]: uhoh someone got the typo flu🙄
```suggestion
Publisher: The ExtremeTuxRacer team
PublisherSupportUrl: https://sourceforge.net/projects/extremetuxracer/support
PackageName: Extreme TuxRacer
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/e/ExtremeTuxRacer/ExtremeTuxRacer/0.8.0.0/ExtremeTuxRacer.ExtremeTuxRacer.locale.en-US.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/e/ExtremeTuxRacer/ExtremeTuxRacer/0.8.0.0/ExtremeTuxRacer.ExtremeTuxRacer.locale.en-US.yaml]: Hahaha, thanks for pointing out.

**Comment by quhxl** [comment]
I should definitely take more breaks...

**Review by OfficialEsco — APPROVED** [review]
:D

**Pull request merged**